test-suite/dune

```diff
@@ -1,3 +1,13 @@
+; The easiest way to generate a portable absolute path is to use OCaml
+; itself to print it
+(executable
+ (name ocaml_pwd)
+ (modules ocaml_pwd))
```

Comment:
Does it make sense for this to live in test-suite? How about putting it in tools/?

Reply:
IMHO is a private binary so it is more robust to put it here so we can move paths around, we could put it in tools and use the new binary scoping feature tho, but I felt it was a bit overkill.